src/library_dylink.js

```diff
@@ -309,14 +314,15 @@ var LibraryDylink = {
         return dynCall(sig, arguments[0], Array.prototype.slice.call(arguments, 1));
       } catch(e) {
         stackRestore(sp);
-        // Exceptions thrown from C++ exception will be integer numbers.
-        // longjmp will throw the number Infinity. Re-throw other types of
-        // exceptions using a compact and fast check.
+        // Exceptions thrown from C++ will be an integer number and longjmp will throw
```

Comment:
`Exceptions thrown from C++ will be pointers (numbers)....`

Reply:
b7e19a9592c5e1eaa2b0e760e2853892cbd1c4e4